PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -708,30 +861,37 @@ imrm ${OutputOrigT2wToT1wPostFS}
 #  so, the primary purpose of the following is to generate the Atlas Registration itself).
 # ------------------------------------------------------------------------------
 
+if [ ! "${T2wInputImages}" = "NONE" ] ; then
+  t2status="RUN"
+else
+  t2status="SKIP"
```

Comment:
In the interest of not having an ever expanding set of flag values, should we perhaps use YES|NO|NONE here again...?

Reply:
Agreed.